common/pipeline_storage/src/test/scala/uk/ac/wellcome/pipeline_storage/elastic/ElasticIndexerTest.scala

```diff
@@ -160,6 +164,7 @@ class ElasticIndexerTest
           val future = indexer(documents)
 
           whenReady(future) { result =>
+            println(result)
```

Comment:
👀

Reply:
Oops, removed in the follow-up PR: https://github.com/wellcomecollection/catalogue/pull/1504/commits/74ea4df83607ec1a9f0275e6de0718277eed3695